stance/main.py

```diff
@@ -30,20 +32,33 @@
         with open("squatterrible_user_skeletons.json", 'rb') as fp:
             user_skeletons = pickle.load(fp)
     else:
+        logger.error("Error parsing command line arguments")
```

Comment:
Why have a different error message?

Reply:
I think we should know what a user tries to put as the video, so I would put that in the log somewhere

Reply:
Because we still throw an exception anyway, which is the intended behavior